app/migrations/0003_tweet.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-10-30 23:14
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0002_auto_20161030_1512'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Tweet',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('text', models.CharField(max_length=256)),
+                ('tweetID', models.IntegerField()),
+                ('created_at', models.CharField(max_length=256)),
+                ('favorite_count', models.IntegerField()),
+                ('lang', models.CharField(max_length=16)),
+                ('location', models.CharField(max_length=256)),
+                ('retweet_count', models.IntegerField()),
+                ('user_name', models.CharField(max_length=256)),
+                ('user_screen_name', models.CharField(max_length=256)),
+                ('user_verified', models.BooleanField()),
+                ('imdbID', models.CharField(max_length=1024)),
```

Comment:
We won't be able to differentiate between different movies with the same name, unfortunately. If we tried to, we'd run into some real issues (ex. who tweets "I really loved Harry Potter and the Deathly Harrows: Part 2"?)